code/modules/clothing/head/helmet.dm

```diff
@@ -78,6 +90,102 @@
 			Smells like sweat."
 	icon_state = "helmet_tan"
 
+/obj/item/clothing/head/helmet/ballistic/shieldfull
+	name = "Blackshield full helm"
+	desc = "Standard military gear. A full-faced variant of the common ballistic helmet. This one bears a few\
+	key upgrades, notably the inclusion of a face-protecting plate and lit eyeshield."
+	icon_state = "bs_fullhelm"
+	item_state = "bs_fullhelm"
+	armor_list = list(melee = 30, bullet = 45,energy = 20, bomb = 30, bio = 0, rad = 0)
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|BLOCKHAIR
+	body_parts_covered = HEAD|FACE|EARS
+	obscuration = LIGHT_OBSCURATION
+	action_button_name = "Toggle Headlamp"
+	brightness_on = 4
+	light_overlay = "bsfullhelm_light"
+
+/obj/item/clothing/head/helmet/ballistic/shieldfull/corpsman
```

Comment:
This meant to be changeable into the base hoods?

Reply:
Nope, the bas hoods don't exist